mobile-security-framework/info.json

```diff
@@ -0,0 +1,701 @@
+        },
+        {
+          "name": "value",
+          "title": "Reference ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify a reference ID to access the attachment metadata from FortiSOAR\u2122's Attachments module. If you have selected Attachment ID, this defaults to the {{vars.attachment_id}} value. If you have selected File IRI, then this defaults to the {{vars.file_iri}} value.",
+          "description": "Specify a reference ID to access the attachment metadata from FortiSOAR\u2122's Attachments module. If you have selected Attachment ID, this defaults to the {{vars.attachment_id}} value. If you have selected File IRI, then this defaults to the {{vars.file_iri}} value."
+        }
+      ],
+      "output_schema": {
+        "hash": "",
+        "status": "",
+        "analyzer": "",
+        "file_name": "",
+        "scan_type": ""
+      }
+    },
+    {
+      "operation": "scan_file",
+      "title": "Scan File",
+      "annotation": "scan_file",
+      "description": "Scan a file which is uploaded on MobSF instance.",
+      "category": "investigation",
+      "is_config_required": true,
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "scan_type",
+          "title": "Scan Type",
+          "type": "select",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the type of file you want to scan.",
+          "description": "Specify the type of file you want to scan.",
+          "options": [
+            "xapk",
+            "apk",
+            "apks",
+            "zip",
+            "ipa",
+            "appx"
+          ]
+        },
+        {
+          "name": "file_name",
+          "title": "File Name",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the name of the app with extension you want to Scan.",
+          "description": "Specify the name of the app with extension you want to Scan."
+        },
+        {
+          "name": "hash",
+          "title": "Hash",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the hash code of the File which you get when uploading a file.",
+          "description": "Specify the hash code of the File which you get when uploading a file."
+        },
+        {
+          "name": "re_scan",
+          "title": "Re Scan File",
+          "type": "checkbox",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Tick if you want to Re scan the File, Default is False.",
+          "description": "Tick if you want to Re scan the File, Default is False.",
+          "value": false
+        }
+      ],
+      "output_schema": {}
```

Comment:
This action does not return any output ?

Reply:
The outputs of the this actions are different for every file and are very large in size, So after discussion it's decided to leave it as blank.